app-electron/src/index.ts

```diff
@@ -143,6 +143,13 @@ async function main() {
       // dock icon is clicked and there are no other windows open.
       if (BrowserWindow.getAllWindows().length === 0) createWindow();
     });
+
+    // Disable secure DNS, as the Portmaster already handles this.
+    // Also, use built-in resolver for better process attribution.
+    app.configureHostResolver({
+      enableBuiltInResolver: false,
+      secureDnsMode: "off"
+    })
```

Comment:
The comment and the setting doesn't match up. "Use built-int resolver" and `enableBuiltInResolver: false`. I guess the comment is wrong here?